.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)